src/cachingStrategies/inMemoryStrategy.ts

```diff
@@ -23,6 +23,12 @@ export class InMemoryStrategy extends CommonCacheStrategyAbstract {
         return this.resultsCacheClient.get(keyWithNamespace);
     }
 
+    public async refreshTtlForCachedResult<T>(namespace: string, key: string, ttl: number, value: CachedResult<T>): Promise<void> {
+        const keyWithNamespace = `${namespace}:${key}`;
+
+        await this.resultsCacheClient.set(keyWithNamespace, value, ttl * 1000);
```

Comment:
I think we don't need to get the value, I suppose if the key is already present will be overrides, right?

Reply:
Otherwise, we can remove the `value` param and based on the key we can get the values and set them again, but on the performance side, I don't know if it is good.

Reply:
Going deeper on this topic, I think I have to get all the previous values (based on the key) and set them again. I'm going to modify the method